.travis.yml

```diff
@@ -25,87 +28,129 @@ before_cache:
   - rm -fv $HOME/.cabal/packages/hackage.haskell.org/01-index.tar
   - rm -fv $HOME/.cabal/packages/hackage.haskell.org/01-index.tar.idx
 
+  - rm -rfv $HOME/.cabal/packages/head.hackage
+
 matrix:
   include:
-    - compiler: "ghc-7.0.4"
-    # env: TEST=--disable-tests BENCH=--disable-benchmarks
-      addons: {apt: {packages: [ghc-ppa-tools,cabal-install-head,ghc-7.0.4], sources: [hvr-ghc]}}
-    - compiler: "ghc-7.2.2"
+    - compiler: "ghc-8.6.1"
+      env: GHCHEAD=true
+      addons: {apt: {packages: [ghc-ppa-tools,cabal-install-head,ghc-8.6.1], sources: [hvr-ghc]}}
+    # Fails due to GHC #15436
```

Comment:
When you say "fails", is it the entire library that fails to build? Just the tests?

If it's the former, we shouldn't list GHC 8.4.3 under `tested-with`. If it's the latter, maybe we should uncomment this and simply enable `TEST=--disable-tests`.

Reply:
Just the tests.

> If it's the former, we shouldn't list GHC 8.4.3 under tested-with. If it's the latter, maybe we should uncomment this and simply enable TEST=--disable-tests.

Sounds good to me.